content/docs/user-guide/gems/reference/environment/vegetation.md

```diff
@@ -0,0 +1,204 @@
+- [Vegetation Layer Spawner](/docs/user-guide/components/reference/docs/user-guide/components/reference/vegetation/layer-spawner)
+- [Vegetation Position Modifier](/docs/user-guide/components/reference/vegetation-modifiers/vegetation-position-modifier)
+- [Vegetation Rotation Modifier](/docs/user-guide/components/reference/vegetation-modifiers/vegetation-rotation-modifier)
+- [Vegetation Scale Modifier](/docs/user-guide/components/reference/vegetation-modifiers/vegetation-scale-modifier)
+- [Vegetation Slope Alignment Modifier](/docs/user-guide/components/reference/vegetation-modifiers/vegetation-slope-alignment-modifier)
+- [Vegetation Altitude Filter](/docs/user-guide/components/reference/vegetation-filters/vegetation-altitude-filter)
+- [Vegetation Distance Between Filter](/docs/user-guide/components/reference/vegetation-filters/vegetation-distance-between-filter)
+- [Vegetation Distribution Filter](/docs/user-guide/components/reference/vegetation-filters/vegetation-distribution-filter)
+- [Vegetation Shape Intersection Filter](/docs/user-guide/components/reference/vegetation-filters/vegetation-shape-intersection-filter)
+- [Vegetation Slope Filter](/docs/user-guide/components/reference/vegetation-filters/vegetation-slope-filter)
+- [Vegetation Surface Mask Depth Filter](/docs/user-guide/components/reference/vegetation-filters/vegetation-surface-mask-depth-filter)
+- [Vegetation Surface Mask Filter](/docs/user-guide/components/reference/vegetation-filters/vegetation-surface-mask-filter)
+
+## Vegetation Areas
+
+The dynamic vegetation system builds around the concept of *vegetation areas* that describe what, where, how, and if vegetation will generate on surfaces. 
+Other systems in the engine register and process Vegetation areas.
+They can examine and take action on points in the world, like placing or preventing a vegetation instance.
+
+In O3DE Editor, vegetation areas are referred to as *vegetation layers*.
+You can add vegetation behaviors to vegetation layers using components that describe, select, reject, or mutate potential vegetation instances. 
+These are called vegetation descriptor providers, selectors, filters, and modifiers respectively. 
+Each has an interface that extends the system with new features. 
+The Vegetation Gem contains several versions of each type that each has a unique purpose and effect on generated vegetation instances. 
+These components expose controls that allow rapid, rule-based, procedural population of any arbitrarily sized part of the world with decorative content. 
+Depending on its configuration, a single Vegetation Layer can act as a small patch of flowers or intelligently blanket the entire world with a convincing variety and groupings of plants and objects.
+
+## Vegetation instances
+
+*Vegetation instances* are the objects placed throughout the world. They're created using vegetation descriptors. 
```

Comment:
```suggestion
*Vegetation instances* are the static objects placed throughout the world. They're created using vegetation descriptors. 
```

Reply:
"Static" is misleading. They're dynamically created and destroyed at runtime, and they can even technically be moving objects. For example, if you put a sphere rigid body into a prefab, you can use the vegetation system to spawn them and they will start rolling around as they appear.